packages/core/src/configuration/Configuration.ts

```diff
@@ -477,7 +470,39 @@ export class Configuration {
    * This setting only works in environments that supports persisted storage.
    * There is also a performance penalty of extra IO/serialization.
    */
-  public usePersistedQueueStorage = false;
+  public usePersistedQueueStorage: boolean = false;
+
+  /**
+   * Gets or sets a value indicating whether to automatically send session start,
+   * session heartbeats and session end events.
+   */
+  public sessionsEnabled = false;
+
+  /**
+   * Internal property used to track the current session identifier.
+   */
+  public currentSessionIdentifier: string | null = null;
```

Comment:
This is internal in the .net client, no modifier for that in ts.

Reply:
I guess that's fine. Not sure what anyone would use this for.